ss-tproxy

```diff
@@ -885,7 +885,9 @@ start_iptables_tproxy_mode() {
     done
 
     if is_enabled_udp; then
-        $1 -t mangle -A SSTP_RULE -p udp -d $direct_dns_ip --dport 53               -j RETURN
+        if ! is_global_mode; then
+            $1 -t mangle -A SSTP_RULE -p udp -d $direct_dns_ip --dport 53 -j RETURN
+        fi
```

Comment:
`When global mode is enabled, everything should go through the proxy.`

In theory, this should be the case, but you forgot to consider one thing: in global mode, I need to support ignlist.ext (the domain name and ip in this will not go through proxy), so I cannot blacklist direct_dns_ip.